src/types.ts

```diff
@@ -74,11 +79,17 @@ export interface TypeDeclrationOptions {
   exportHelper?: boolean
 }
 
-export interface InjectImportsOptions {
+export interface AddImportsOptions {
   /**
    * @default false
    */
   mergeExisting?: boolean
+
+  /**
+   * Inject using `import` or require
+   * @default auto-detection
+   */
+  isCJS?: boolean
```

Comment:
Why we would need this feature?

Reply:
Was trying to have the resolve logic inside the utils, removed.